packages/web-components/src/components/search/search.ts

```diff
@@ -8,7 +8,7 @@
  */
 
 import { customElement } from 'lit/decorators.js';
-import BXSearch from '../../internal/vendor/@carbon/web-components/components/search/search.js';
+import CDSSearch from '../../internal/vendor/@carbon/web-components/components/search/search.js';
```

Comment:
i think we might want to keep it as `BXSearch` for now until we update `web-components` to v2 since it's still pulling from `@carbon/web-components: 1.xx`

Reply:
Completely missed this while I was working on the changes yesterday, but changed it back, good call!